promtimer/promtimer.py

```diff
@@ -219,6 +220,12 @@ def main():
                         help='don\'t open browser tab automatically on start')
     parser.add_argument("--verbose", dest='verbose', action='store_true',
                         default=False, help="verbose output")
+    parser.add_argument('--refresh', dest='refresh',
+                        help='grafana refresh interval; '
+                             'only valid when connecting to live cluster')
+    parser.add_argument('--nodes', dest='nodes', 
```

Comment:
I think this should be able to be solved using alternate address - which the Capella clusters must be provisioned with if they are accessed via a NAT. Basically we need to change the logic in ServerNode.get_stats_sources to not use the regular name of the node but use the alternate names. 

Then we might need to provide a global argument to promtimer to tell it to use alternate addresses of the nodes (or the network=external) in SDK terminology.)

Reply:
That would work for people who has direct access to the cluster i.e customers, user and for our development environments.  

For support that is not how we get access, basically we end up doing a port forwarded. This is why it was done like this.

Reply:
To be clear we should add SRV support :)

Reply:
Added nargs